docs/examples/react-native/RNThemisExample/android/app/build.gradle

```diff
@@ -0,0 +1,123 @@
+apply plugin: "com.android.application"
+apply plugin: "com.facebook.react"
+
+/**
+ * This is the configuration block to customize your React Native Android app.
+ * By default you don't need to apply any configuration, just uncomment the lines you need.
+ */
+react {
+    /* Folders */
+    //   The root of your project, i.e. where "package.json" lives. Default is '..'
```

Comment:
it's our comments or generated by some tool?

Reply:
It's generated.